deb-arm/Dockerfile

```diff
@@ -112,6 +120,21 @@ RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
 ENV PATH="/usr/local/go/bin:${PATH}"
 ENV PATH="${GOPATH}/bin:${PATH}"
 
+# Rust is needed to compile the SDS library
+RUN if [ "$DD_TARGET_ARCH" = "aarch64" ] ; then curl -sSL -o rustup-init https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${RUSTUP_ARCH_AARCH64}/rustup-init \
+    && echo "${RUSTUP_SHA256_AARCH64}  rustup-init" | sha256sum --check \
+    && chmod +x ./rustup-init \
+    && ./rustup-init -y --profile minimal --default-toolchain ${RUST_VERSION} \
+    && echo "${RUSTC_SHA256_AARCH64}  $HOME/.cargo/bin/rustc" | sha256sum --check \
+    && rm ./rustup-init; fi
```

Comment:
Just double checking, does this leave some kind of temporary files behind? We had a few GB wasted by go temporary files in other build images, I'd rather not have to download those for rust :sweat_smile:

Reply:
Nah as long as we only ship the final binaries and not all the intermediary objects I think we're good :)

Reply:
For a future reader: Correction 14MB per binary and as they're hard-links, they only count once for the inode size.
The toolchain is 800MB (most of it are the `.so` _used_ by the Rust toolchain), there won't be much I can do.